tests/shrub/v3/test_shrub_service.py

```diff
@@ -0,0 +1,78 @@
+"""Unit tests for shrub_service.py."""
+
+import json
+import pytest
+
+from shrub.v3.evg_task import EvgTask, EvgTaskDependency
+from shrub.v3.evg_build_variant import BuildVariant, DisplayTask
+from shrub.v3.evg_command import FunctionCall
+from shrub.v3.evg_project import EvgProject
+from shrub.v3.shrub_service import ShrubService
+
+
+@pytest.fixture
+def project():
+    n_tasks = 10
+
+    def define_task(index):
+        name = f"task_name_{index}"
+
+        return EvgTask(
+            name=name,
+            commands=[
+                FunctionCall(func="do setup"),
+                FunctionCall(
+                    func="run test generator",
+                    vars={"parameter_1": "value 1", "parameter_2": "value 2"},
+                ),
+                FunctionCall(func="run tests"),
+            ],
```

Comment:
Add a command/field to test multiline string formatting, which are likely used with `shrub.v3.evg_command.subprocess_exec` and `shrub.v3.evg_command.shell_exec` (among other cases).

Also add a command/field to test quoting behavior (for values that would otherwise be treated as another type, e.g. boolean and numeric values).

Reply:
Done